cypress/integration/plugins/observability-dashboards/3_trace_analytics_traces.spec.js

```diff
@@ -6,6 +6,10 @@
 /// <reference types="cypress" />
 
 import { setTimeFilter, TRACE_ID } from '../../../utils/constants';
+import customParseFormat from 'dayjs/plugin/customParseFormat';
+import dayjs from 'dayjs';
```

Comment:
Do we need to backport all the changes in this file? Seems `dayjs` not been used in test cases below.